test/pytorch_test_base.py

```diff
@@ -208,7 +208,6 @@
     'TestNNDeviceTypeXLA': {
         'test_embedding_backward',  # sparse
         'test_embedding_dense_grad',  # slow
-        'test_EmbeddingBag_per_sample_weights_and_new_offsets',  # FIXME! UndefinedTensorImpl::_singleton
```

Comment:
Btw there're 3 more embeddingbag tests below, could you confirm they're still failing (are they different issues?) Thanks!

Reply:
Yes, they are still failing. Two of them are failing because they test sparse tensor behavior. The other fails due to a different runtime error.

Reply:
Would you mind updating them with the latest error message? (The singleton issue should be fixed by your PR) Thanks

Reply:
Done. And had a nice surprise of finding an extra test that's passing but missed before